source/_integrations/schlage.markdown

```diff
@@ -37,6 +37,12 @@ Once you have enabled the Schlage integration, you should see the following bina
 
 - **Keypad disabled** - Indicates that the keypad has been disabled, typically due to too many incorrect lock codes being attempted.
 
+## Select
+
+Once you have enabled the Schlage integration, you should see the following selects:
+
+- **Auto Lock Time** - Configure the time until the deadbolt automatically locks, or disable the auto-lock feature entirely.
```

Comment:
```suggestion
- **Auto Lock Time** - Configure the time until the deadbolt automatically locks, or disable the auto-lock feature entirely. For example, `0` means auto
```
```suggestion
- **Auto Lock Time** - Configure the time until the deadbolt automatically locks, or disable the auto-lock feature entirely. For example: `0`: auto-lock is disabled, `15`: it auto-locks after 15 seconds, `300`:  it auto-locks after 5 minutes.
```
Maybe an example of some possible values would be helpful here? Could you check if the suggestion is correct?